core/domain/exp_jobs_one_off.py

```diff
@@ -325,7 +325,10 @@ def map(item):
     @staticmethod
     def reduce(key, values):
         if key == 'SUCCESS':
-            yield (key, values[:VALID_MATH_INPUTS_YIELD_LIMIT])
+            if values[0] == 'None':
```

Comment:
No -- I mentioned this already in chat. Use a different key for this case.

Reply:
Done.